subset/security/.htpasswdpass

```diff
@@ -0,0 +1 @@
+root:$apr1$gFVTHyqx$RIoKjePaNJv1FEmI1LAEo0
```

Comment:
potentially can these be generate during the docker build ? or the start_faux script ?